Xwt.XamMac/Xwt.Mac/DialogBackend.cs

```diff
@@ -169,6 +169,13 @@ public void RunLoop (IWindowFrameBackend parent)
 		{
 			Visible = true;
 			modalSessionRunning = true;
+
+			if (parent.Window is NSWindow nsParent)
+            {
+				nsParent.AddChildWindow(this, NSWindowOrderingMode.Above);
+				Util.CenterWindow(this, nsParent);
```

Comment:
Don't we also want to center when not parented?

Reply:
fixed